### ISSUE TITLE
jshowoff.css upsets my page layout

### ISSUE THREAD

**Issue by justinUK** [ekallevig/jShowOff#19]
Hi
I've just installed jshowoff on our oriental carpets sites here..  http://www.orientalrugsonline.co.uk/ which looks great.  I've got one weird effect which I'm pretty sure is caused by use of jshowoff.css.  Problem is that the whole page has moved down approximately 25pixels in Mozilla and about 50 in IE (why the difference?).  I'm no expert in css and I can't immediately tell which bits I need and which I don't -- I think I just need to delete the stuff I don't to get this working properly. My version of the css file is here http://www.orientalrugsonline.co.uk/jshowoff/jshowoff.css 
Thanks in advance for any help.

**Comment by shashankind** [comment]
I am also facing page layout problem.

**Comment by karloscarweber** [comment]
You're right on Justin. The plugin includes styling for the demo page in the jshowoff.css file. These often conflict with your sites current styles. You should be good if you delete the demo styles from from the css file.

**Comment by justinUK** [comment]
Thanks for your reply. I think I finally worked it out when I was doing 
it and here's the result.....
http://www.orientalrugsonline.co.uk/
I would have quite liked the random feature that's trailed as a future 
enhancement but I think no further development is taking place.
Justin

**Comment by karloscarweber** [comment]
If you're pulling the rugs from a database then you could randomize them on that end. That's how I would do it.
